src/rgw/rgw_rados.h

```diff
@@ -2125,6 +2130,99 @@ struct tombstone_entry {
 
 class RGWIndexCompletionManager;
 
+class CacheRequest {
+public:
+  Mutex lock;
```

Comment:
this is a Ceph Mutex, which is gradually being replace by std::mutex;  if you aren't working with an interface which actually takes a Mutex, use std::mutex